src/app/(views)/managed-courses/[courseId]/page.tsx

```diff
@@ -15,6 +48,7 @@ export default function Course() {
   const course = courses[0]
   const [deletingCourse, setDeletingCourse] = useState(false)
   const [deletingModule, setDeletingModule] = useState(false)
+  const [visible, setVisible] = useState(false)
```

Comment:
Use a more descriptive name. What is visible? maybe const [showDeleteModa, setShowDeleteModal]?